### ISSUE TITLE
about training loss

### ISSUE THREAD

**Issue by shanyun123456** [HRNet/DEKR#12]
The loss of training is very small and  decreases slowly  when i finetune model on cocodatasets. Is't normal？
and how many epoches do you set?

**Comment by shanyun123456** [comment]
taining from scatch on cocodatasets, not finetune

**Comment by Gengzigang** [comment]
We used 140 epochs. You can find the settings that we used exactly in the experiments folder.

**Comment by shanyun123456** [comment]
what about the loss of training

**Comment by Gengzigang** [comment]
The heatmap loss is usually on the order of 1e-4 and the offset loss is usually on the order of 1e-5.

**Comment by shanyun123456** [comment]
Thanks,
By the way, What the loss of training did you get when you finish the 140 epochs of training?

**Comment by Gengzigang** [comment]
For the model with HRNet-W48 backbone, the AP on COCO validation with multi-scale testing is 72.3, the heatmap loss is 1.752e-4 and the offset loss is 4.764e-5.

**Comment by shanyun123456** [comment]
ok , thank you!
